.github/workflows/checks.yml

```diff
@@ -1,9 +1,7 @@
 name: Checks
 
 env:
-  STARKNET_VERSION: "0.13.0"
-  RPC_SPEC_VERSION: "0.6.0"
-  DEVNET_SHA: "1bd447d"
+  DEVNET_SHA: "c6ffb99"
```

Comment:
shouldn't we use devnet version: 230b74c? https://github.com/0xSpaceShard/starknet-devnet-rs/releases/tag/v0.0.3

Reply:
It doesn't make any difference, as there is no relevant changes in the commits following `c6ffb99` (only changes to CI and package version).